proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -55,6 +55,10 @@ message Repository {
   // url is the user configurable URL in the description of the repository,
   // always included the scheme and will not have a #fragment suffix.
   string url = 12;
+  // default_branch in a BSR repository. It is used when syncing a git repository, to make sure both
+  // default branches (BSR and Git) are in sync. By default, every BSR repository is created with a
```

Comment:
What happens if the default branches (BSR and Git) are out of sync? Would it reject the sync until the user manually changes the default branch in BSR to match?

Reply:
That's correct, but that's in the `buf sync` scope, in an upcoming PR.

Reply:
Here's that PR: https://github.com/bufbuild/buf/pull/2328